_episodes/16.md

```diff
@@ -0,0 +1,367 @@
+---
+layout: episode
+title:  "16: Make data fluid with Apache Druid"
+date: 2021-04-29
+tags: druid apache realtime analytics sql
+youtube_id: "qS8XB4zHFHY"
+wistia_id: "r2b09jbb3h"
+sections: 
+   - title: "Concept of the week"
+     desc: "Apache Druid and realtime analytics"
+     time: 891
+   - title: "PR of the week"
+     desc: "PR 3522 Add Druid connector"
+     time: 2015
+   - title: "Demo"
+     desc: "Using the Druid Web UI to create an ingestion spec querying via Trino"
+     time: 3689
+   - title: "Question of the week"
+     desc: "Does the Druid connector?"
+     time: 4220
+---
+
+Trino nation, We want to hear from you! If you have a question or pull request 
+that you would like us to feature on the show please join the 
+[Trino slack](slack.html), and go to the \#trino-community-broadcast channel 
+and let us know there. Otherwise, you can message Manfred Moser or Brian Olsen 
+directly. Also, feel free to reach out to us on our Twitter channels Brian 
+[@bitsondatadev](https://twitter.com/bitsondatadev) and Manfred 
+[@simpligility](https://twitter.com/simpligility).
+
+If you want to show us some 💕, please [give us a ⭐ on GitHub](/star).
+
+<p align="center">
+<img align="center" width="50%" height="100%" src="/assets/episode/16/trino-druid.png"/><br/>
+Commander Bun Bun the speedy druid!
+</p>
+
+## Iceberg links
+
+ * [Apache Druid](https://druid.apache.org/)
+ * [Apache Druid Community](https://druid.apache.org/community/)
+ * [Druid Forum](https://www.druidforum.org/)
+
+## Guests
+ * Samarth Jain, Software Engineer at Netflix 
+ ([@samarthjain11](https://www.linkedin.com/in/samarthjain11/))
+ * Parth Brahmbhatt, Senior Software Engineer at Netflix 
+ ([@brahmbhattparth](https://twitter.com/brahmbhattparth/))
+ * Rachel Pedreschi, VP Community and Developer Relations at 
+ [Imply](https://imply.io/) ([@rachelpedreschi](https://twitter.com/rachelpedreschi))
+
+## Release 356
+
+Release notes discussed: <https://trino.io/docs/current/release/release-356.html>
+
+ - General:
+   - MATCH_RECOGNIZE - query over range of record with trending and such
```

Comment:
- MATCH_RECOGNIZE clause support, used to detect patterns in a set of rows within a single query